packages/web/src/executors/file-server/schema.json

```diff
@@ -61,6 +61,11 @@
         }
       },
       "additionalProperties": true
+    },
+    "spa": {
+      "type": "boolean",
+      "description": "Serve `SPA` app.",
```

Comment:
This should probably detail the differences, rather than just reiterating the option name. Something like: `Redirect not-found errors to index.html (useful for SPAs)`.